Content.Server/_NF/GameRule/PointOfInterestSystem.cs

```diff
@@ -23,11 +23,11 @@ public sealed class PointOfInterestSystem : EntitySystem
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
     [Dependency] private readonly IRobustRandom _random = default!;
     [Dependency] private readonly IConfigurationManager _configurationManager = default!;
```

Comment:
I'm pretty sure C comes before P
also shorten the names

Reply:
unexpected kurwaball